src/shared/components/header/index.js

```diff
@@ -23,14 +23,18 @@ class Header extends Component {
 
     return (
       <header className={ classNames(styles.header, className) }>
-        <Observer className={ styles.target } onChange={ this.handleObserverView } />
+        <Observer className={ styles.target } onChange={ this.handleObserverView }>
+          <span />
```

Comment:
Why this `</span>` here?

Reply:
Because the new `Observer` lib that we have requires to have a child note